mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -176,73 +176,6 @@ computeCopyPerThread(Type elementType, int64_t copyPerThread, int64_t kPerBlock,
   return std::make_pair(copyKPerThread, copyDPerThread);
 }
 
-/// Applies the transforms that take a G x K x D matrix to a k_iter x bid x tid
-/// x iter value suitable for using in a global load loop. `dName` should be "m"
-/// and "n", and is used to make the maps have the right names for debugging.
-static FailureOr<Value> wrapMatrixForGlobalLoad(
-    OpBuilder &b, Location loc, Value matrix, StringRef dName,
-    ArrayRef<StringRef> bidGridOrder, ArrayRef<int64_t> bidGridLengths,
-    int64_t gridSize, int64_t blockSize, int64_t kPerBlock, int64_t dPerBlock,
-    int64_t kPerThread, int64_t dPerThread, GemmDimension vectorDim) {
-
-  if (dName != "m" && dName != "n") {
-    return emitError(loc, "expected dName to be m or n but got " + dName);
-  }
-
-  StringRef thisBlockDim = dName == "m" ? "m_block" : "n_block";
-  StringRef otherBlockDim = dName == "m" ? "n_block" : "m_block";
-
-  MemRefType matrixType = matrix.getType().cast<MemRefType>();
-  ArrayRef<int64_t> matrixShape = matrixType.getShape();
-  int64_t kGlobal = matrixShape[1];
-  int64_t dGlobal = matrixShape[2];
-
-  int64_t kIters = kGlobal / kPerBlock;
-  int64_t dataPerThread = (kPerBlock * dPerBlock) / blockSize;
-
-  SmallString<8> dIterName = llvm::formatv("{0}_iter", dName);
-  SmallString<8> dThreadName = llvm::formatv("{0}_thread", dName);
-
-  // Note: (kThreads * dThreads) = (kPerBlock * dPerBlock) / dataPerThread) =
-  // blockSize
-  int64_t kThreads = kPerBlock / kPerThread;
-  int64_t dThreads = dPerBlock / dPerThread;
-
-  TopDownTMBuilder splitId(
-      b, {"k_loop", "g_block", "m_block", "n_block", "tid", "iter"},
-      {kIters, bidGridLengths[0], bidGridLengths[1], bidGridLengths[1],
```

Comment:
In my analysis (mostly IR diffs), I have fixed this bug currently; we use m_blocks as n_blocks.